extensions/podman/packages/extension/src/configuration/registry-configuration.ts

```diff
@@ -0,0 +1,91 @@
+/**********************************************************************
```

Comment:
new folder <3 thanks for taking https://github.com/podman-desktop/podman-desktop/issues/10327 into consideration